sdk/servicebus/service-bus/README.md

```diff
@@ -13,6 +13,7 @@ Use the client library for Azure Service Bus in your Node.js application to
 
 [Source code for v1.1.5](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/service-bus_1.1.5/sdk/servicebus/service-bus) |
 [Package for v1.1.5 (npm)](https://www.npmjs.com/package/@azure/service-bus/v/1.1.5) |
+[API Reference Documentation for v1.1.5](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/1.1.5/index.html) |
```

Comment:
We can update this once the docs.microsoft.com gets updated.(in a couple of days)

Reply:
Can we have this point to docs.microsoft.com right away?
The above link shows some weird entries on the right
<img width="413" alt="Screen Shot 2020-03-27 at 9 39 57 PM" src="https://user-images.githubusercontent.com/16890566/77815001-8bf72000-7073-11ea-9de4-3d5063804fef.png">

Reply:
I believe those are from the constants.ts file.
Nonetheless, here's the PR #8098 to update to docs.microsoft.com